doc/man1/openssl.pod

```diff
@@ -789,34 +796,28 @@ OpenSSL command to generate an alternative chain.
 
 =over 4
 
-=item B<-xchain_build>
-
-Specify whether the application should build the certificate chain to be
-provided to the server for the extra certificates via the B<-xkey>,
-B<-xcert>, and B<-xchain> options.
-
```

Comment:
Why is this removed?

Reply:
I anticipated this question :smile: 
I removed this part because a duplicate of it had crept in before.

Reply:
Ah, ok